tools/scripts/lint_check/IWYU.md

```diff
@@ -0,0 +1,9 @@
+使用IWYU报错后有如下一些原则：
```

Comment:
translate to English or just remove this file

Reply:
Done